models/tridimensional/docking_validation/join_csv_results.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import os.path
+import fnmatch
+import datetime
+import csv
```

Comment:
I think the convention we've been using is put the imports in alphabetical order